src/index.tsx

```diff
@@ -114,7 +115,13 @@ export async function init(environment = "production") {
     } else {
       ReactDOM.render(
         <ErrorBoundary>
-          <App update={update} fiddle={fiddle} embeddingParams={embeddingParams} windowContext={appWindowContext}/>
+          <App
+            update={update}
+            fiddle={fiddle}
+            embeddingParams={embeddingParams}
+            windowContext={appWindowContext}
+            quickStart={quickStart}
```

Comment:
Should `quickStart` be part of `windowContext` or `embeddingParams`?

Reply:
I prefer a top level parameter to make the url look as readable as possible